harvest-agent-h3/src/main/java/org/webcurator/core/harvester/agent/HarvesterH3.java

```diff
@@ -825,61 +842,5 @@ public ScriptResult executeShellScript(String jobName, String engine, String she
         return heritrix.ExecuteShellScriptInJob(jobName, engine, shellScript);
     }
 
-    public static void main(String[] args) {
```

Comment:
I'm assuming this block of code was removed because it's not used.

Reply:
Correct, it was just for testing